src/pages/tutorials/aiops/scopeBasedGrouping/index.mdx

```diff
@@ -122,7 +122,7 @@ NOTE: The Priority specifies the order in which the Policies will be tried. Once
 
 ## 7-5: Inject some sample Location events
 
-Create super_group_events.sh using below script to insert sample events.
+Create super_group_events.sh using below script to insert sample events. Please ensure you are logged into your probe server (pi-template) and your current OCP project is "noi" before executing the scripts below.
```

Comment:
Same as above, let's remove reference to pi-template.

Reply:
Removed reference to pi-template